Marlin/Marlin_main.cpp

```diff
@@ -5307,7 +5335,17 @@ void home_all_axes() { gcode_G28(true); }
             for (float circles = -offset_circles ; circles <= offset_circles; circles++) {
               const float a = RADIANS(180 + 30 * axis),
                           r = delta_calibration_radius * (1 + circles * (zig_zag ? 0.1 : -0.1));
-              z_at_pt[axis] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1, false);
+              #if !ENABLED(PROBE_MANUALLY)
+                z_at_pt[axis] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1, false);
+              #else
+                _man_probe_pt(cos(a) * r, sin(a) * r);
+                KEEPALIVE_STATE(PAUSED_FOR_USER);
+                wait_for_user = true;
+                while (wait_for_user) idle();
+                wait_for_user = false;
```

Comment:
The nozzle cannot be moved during this loop, so if this is meant to get a current Z position supplied by user input, I don't see how it's supposed to work.